Automattic/themes
# Modern Business Full Site Editing  - add FSE classes to header and footer

#### Changes proposed in this Pull Request:

Add  `.fse-enabled` selectors to block css for header and footer if FSE is enabled in order to apply the correct front end block styles to the FSE header and footer content.

#### Testing
- Build modern-business theme and copy or link to local FSE install
- Checkout https://github.com/Automattic/wp-calypso/pull/35209 in your FSE environment (this is needed in order to apply the `.fse-enabled` class to the body element
- Add a Paragraph block to the Header and Footer template parts and set the background colour to one of the primary colours
- View the page in the front end and check that the blocks have a background colour - this should be due the existence of a either `.fse-template-content`  on the `site-header` and `.site-footer`  divs if testing first commit, or `fse-enabled` class on body if testing second commit.
- Disable the FSE plugin and check that the `.fse-template-content` class is no longer present on the `.site-header` and `.site-footer` divs if testing first commit, or `fse-enabled` not on body if testing second commit

**Before**

<img width="534" alt="before" src="https://user-images.githubusercontent.com/3629020/62341603-1369af00-b538-11e9-8108-95633455e74d.png">

**After**

<img width="519" alt="after" src="https://user-images.githubusercontent.com/3629020/62341613-1cf31700-b538-11e9-9d65-f6e237322f8e.png">

Fixes https://github.com/Automattic/wp-calypso/issues/35058
